### PR TITLE
Disble autocreate and automate schema into docs

### LINKED ISSUE #413: Document advice for keyspace/table auto-creation in README
> And document that the auto creation of keyspace and tables is mostly intended as a convenience for development and tests. For production it's better to create the tables before starting the application so that all needed configuration can be specified.

_Originally posted by @patriknw in https://github.com/akka/akka-persistence-cassandra/issues/148#issuecomment-289263428_

### LINKED ISSUE #582: Clarify schema setup in documentation
The schema is printed in the documentation or you can set `tables-autocreate` but overall, ops-related topics are not really discussed in the documentation.

There should be section that details how the usual installation and maintenance (migration) operations should take place and also what kind of compatibility guarantees the plugin gives in general (e.g. no migrations necessary for major/minor versions).

Also the schema printed in the documentation should be exactly the same as the one in the code and not a copy.

### LINKED ISSUE #645: disable autocreate by default
Because we strongly recommend against it for production, and if someone accidentally use it in production with the default settings for replication it can be a disaster.

### PR DESCRIPTION
**Pull request by chbatey**
Fixes #645
Fixes #582
Fixes #413

### CODE REVIEW

**Comment by patriknw** [comment]
this will have some conflicts with https://github.com/akka/akka-persistence-cassandra/pull/649

File: build.sbt [hunk 1]
```diff
@@ -17,6 +17,9 @@ lazy val session = (project in file("session"))
     OsgiKeys.importPackage := Seq(akkaImport(), optionalImport("org.apache.cassandra.*"), "*"),
     OsgiKeys.privatePackage := Nil)
 
+lazy val dumpSchema = taskKey[Unit]("Dumps cassandra schema for docs")
+dumpSchema := (core / runMain in (Test)).toTask(" akka.persistence.cassandra.PrintCreateStatements").value
```
Comment (johanandren) [inline build.sbt]: Neat but will run it for every build of the docs, perhaps that is ok, not sure there is a super easy way to use the sbt caching infra in a task though.

File: docs/src/main/paradox/journal.md [hunk 2]
```diff
@@ -34,88 +34,25 @@ It can't be turned on by the plugin as it is in the driver's reference.conf and
...
-CREATE TABLE IF NOT EXISTS akka.metadata(
-  persistence_id text PRIMARY KEY,
-  deleted_to bigint,
-  properties map<text,text>);
+CREATE KEYSPACE IF NOT EXISTS akka WITH replication = {'class': 'NetworkTopologyStrategy', '<your_dc_name>' : 3 }; 
 ```
+
+For local testing, and the default if you enable `cassandra-plugin.journal.keyspace-autocreate` you can use the following:
+
+@@snip [journal-schema](/target/journal-keyspace.txt) { #journal-keyspace }
```
Comment (johanandren) [inline docs/src/main/paradox/journal.md]: I guess you verified that this actually works right?

Comment (chbatey) [reply docs/src/main/paradox/journal.md]: Yep!

File: build.sbt (hunk 1 above)
Comment (chbatey) [reply build.sbt]: I think that is okay for now

**Pull request merged**